### ISSUE TITLE
deletion of wrong locations in the guis

### ISSUE THREAD

**Issue by SlrG** [owntracks/talk#152]
In the last weeks I started to have a number of wrong location recordings from my android phone. As I am from Germany, I assure you I was not in Sweden in the middle of the night, even if owntracks thinks otherwise. ;)

![grafik](https://user-images.githubusercontent.com/978967/208987822-d71afc9b-ff48-41ec-901f-5bae7dbdc868.png)

I know it is possible to remove such wrong information, by manually editing the .rec files. This was not much of a problem, when I never had those wrong recordings. But now I think it would be great if I could delete those points from the frontends. Maybe place a small delete button in the information window shown in my screenshot, that pops up when selecting a single location?

This would be a cool privacy addition, too. One could easily delete a location if it should not appear in the recordings any longer.

Ideally this should be possible in all frontends/apps. For this to happen, the recorder web view would have to allow to select a single location like the frontend does already and display the same information window with the new deletion button.

IMHO (but this is probably a lot of work and would make the frontend somewhat obsolete) the mobile apps should allow to show not only the current location, but a configurable time span of recorded locations, too. This would allow the user to select single locations, display their information and remove them, if desired.

**Comment by lnxd** [comment]
I found this Issue looking for the same feature. In my opinion, it would alternatively (additionally?) be useful to be able to permanently "hide" records from the GUI. For people who use this for legal purposes for example, so that the record wasn't manually edited.

I only set things up yesterday, but I'm currently checking to see if it's possible to reject data points where the acceleration speed is above a certain value eg. 1km/sec to reduce instances like this.

My iPhone registered 4 different locations today in the same second within the same city while I was in IKEA:

<img width="1364" alt="image" src="https://github.com/owntracks/talk/assets/48756329/c847d133-e6c5-40c4-ba7a-f3dce6b576cc">

Unlike @SlrG I can assure anyone that reads this that I was actually in 4 places simultaneously, but it would be useful to be able to clean up the map.
